REVIEW.md

```diff
@@ -0,0 +1,85 @@
+# Review process
+> This is a general process, a project specific process must be documented in its own repository.
+
+We rely on peer review as much as possible :
+> https://en.wikipedia.org/wiki/Peer_review
+
+Try to be as benevolent as possible when reviewing your peers. 
+Someone theorise *Egoless programming* that fits our overall philosophy quite well.
+> https://en.wikipedia.org/wiki/Egoless_programming
+
+## Branch
+When starting to work on a subject on any project, the first thing to do is creating your own branch.
+The base branch should be `master` and the new branch should reflect the subject you are working on:
+- a new feature must be prefixed by `feat/` ex: `feat/new-contribution-form`
+- a bug fix can be prefiexed by `fix/`
+Or `chore`, `docs` or any other word you see fit!
+
+> We use Conventionnal commits for commit message, and this branch naming is inspired by the keywords use there
+
+## Commits
+For commit messages and we use Conventionnal commits convention.
+Beware, a single commit shouldn't break anything (nor the test, nor the app, nor anythign.
+We should be able to checkout any commits and run the app without any issues.
+
+> When unsure just squash everyhting.
+
+## History
+Keep your history clean! There can be many commits or only one, as long as the story told is consistent.
+If you the history doesn't feel right, just be a negationist and use (and abuse) `git rebase -i`. 
+Just as long as you don't reright other people's history it's fine (really).
+
+And if you feel like, feel freer to `squash` your branch history into a single commit.
+
+## Pull/Merge request
+When ready you can open a Pull Request targeting the main branch.
+> BTW shall we rename `master` to `main`?
```

Comment:
Yes!

Reply:
Could you create issues in the respective repositories?
I'm curious to see how you would fill in the "feature request template" for this.